app/src/components/OrderBookSnapshot.tsx

```diff
@@ -256,6 +265,110 @@ class OrderBookSnapshot extends Component<Props, State> {
         }
     };
 
+    /**
+     * @desc Handles changes to orderbook from playbackModifications data
+     */
+    handlePlaybackModifications = (playbackData: PlaybackData) => {
+        const { listItems } = this.state;
+        const { onTimestampSelected, enqueueSnackbar } = this.props;
+        const selectedTimestampInfo: SelectedTimestampInfo = {
+            currentOrderbookTimestamp: playbackData.timestamp,
+            lastModificationType: LastModificationType.GRAPH,
+        };
+        onTimestampSelected(selectedTimestampInfo);
+        const modificationsLength = playbackData.modifications.length;
+        let ctr: number = 0;
+        const data = processOrderBookPlayback(listItems);
+        let { newListItems } = data;
+        const { newMaxQuantity } = data;
+        while (ctr < modificationsLength) {
+            const playbackModification = playbackData.modifications[ctr];
+            const { price } = playbackModification;
+            const { from } = playbackModification;
+            const { to } = playbackModification;
```

Comment:
extraction can be done in a single line ? `{price, from, to} = playbackModification`

Reply:
👍